README.md

```diff
@@ -64,7 +64,7 @@ console.log(faking.f64());
 ### Unsupported Seeded Generation
 The following data generators do not support seeded generation:
 
-- [UUIDs](src/data/defaults/uuids.rs), any.
+- [UUIDs](src/data/defaults/uuids.rs), ([v4](src/data/defaults/uuids.rs#L28) | [v7](src/data/defaults/uuids.rs#L48)).
```

Comment:
v4 and v7 should be possible to made consistent using the uuid builder 
https://docs.rs/uuid/latest/src/uuid/builder.rs.html#583-587
https://docs.rs/uuid/latest/src/uuid/v7.rs.html#47-53